app/styles/helpers/inverse-element.less

```diff
@@ -23,13 +41,17 @@
   .upf-select,
   .selectize-input {
     color: @color-text-contrast;
-    background-color: @upf-primary-rock-blue;
-    border-color: @upf-primary-rock-blue;
+    background-color: fade(white, 10%);
```

Comment:
You can use `upf-primary-white` instead of `white`. I know it's exactly the same but using variables will help us a lot when we will get rid of unused stuffs